test/widgets/content_test.dart

```diff
@@ -161,4 +161,33 @@ void main() {
       check(tester.takeException()).isA<AssertionError>();
     });
   });
+
+  group("CodeBlock", () {
```

Comment:
nit: move this above the two existing groups

That way we keep the tests in the same order as the code they're testing, at least at a coarse level of granularity: CodeBlock/CodeBlockNode, then LinkNode which is handled by InlineContent, then RealmContentNetworkImage.

(That isn't always possible because a test doesn't always correspond neatly to a particular area of code. But in this case they do nicely correspond, so we can.)

Reply:
ack